src/ORM/Connect/DBSchemaManager.php

```diff
@@ -28,6 +28,15 @@ abstract class DBSchemaManager
      */
     private static $check_and_repair_on_build = true;
 
+    /**
+     * @config
+     * For large database, you can exclude a list of table names for checking and rapairs
+     * In this way smaller tables are still checked for consistency and health.
+     *
+     * @var array
+     */
+    private static $exclude_tables_for_checks = [];
```

Comment:
```suggestion
     * For large databases you can declare a list of table names which will be excluded from CHECK TABLE and REPAIR TABLE queries during dev/build.
     * Only use this configuration if you know what you are doing and have identified specific tables as being problems during your dev/build process.
     */
    private static array $exclude_tables_for_checks = [];
```
`@config` isn't necessary and we had actually planned to explicitly remove it in CMS 5 if we had time. It will most likely be removed as an annotation in some future update - all `private static` properties are to be interpreted as config unless the `@internal` attribute is used.

We also prefer typehints over phpdoc typing for new properties.

Also changed the comment to be a little more explicit and to warn off flippant use of this config.